src/systolic_array/unit_test/Makefile

```diff
@@ -0,0 +1,30 @@
+.PHONY: all test clean
```

Comment:
Is it possible to use SCons instead?

Regardless, we still need to make this test run automatically by CI.

Reply:
It must be possible, it's just that I don't know SCons enough for how to do so.. (like how to generate another binary plus gem5.opt)

I put the test in the CI config so it will tested in the CI build.

Reply:
SGTM. This reminds me - we have never run the gem5 unit/integration tests in our CI. We may want to do that at some point :)